configtx/orderer/orderer.go

```diff
@@ -22,12 +22,14 @@ const (
 	ConsensusTypeSolo = "solo"
 
 	// ConsensusTypeKafka identifies the Kafka-based consensus implementation.
+	// Deprecated: ConsensusTypeKafka is deprecated
```

Comment:
// Deprecated: the kafka consensus type is no longer supported.

change all "Deprecated:" comments to this language, there is no need to restate the name of the variable.

Reply:
done